test/plugins/test_unittest_assert_check.py

```diff
@@ -0,0 +1,101 @@
+"""Test unittest_assert_check.py"""
+
+import pytest
+
+from .pylint_test import run_pylint
+
+
+def test_bad_asserts():
+    source = """\
+        import unittest
+
```

Comment:
maybe move this to a `.py` file and provide a path to that file instead

Reply:
This is the trend in entire `edx-lint/test/plugins/` directory.

https://github.com/edx/edx-lint/tree/38a1b8580e8621629422cf7fd57d7399c9587bf0/test/plugins

Reply:
it shouldn't be hard to read from a file instead, it'd be a lot cleaner solution

Reply:
An advantage of keeping the code in a string is that I can read and understand the whole test without having to go find another file.

Reply:
I have moved the test code back to strings